fusor-ember-cli/app/routes/review/progress/overview.js

```diff
@@ -3,12 +3,19 @@ import Ember from 'ember';
 export default Ember.Route.extend({
   model: function() {
     var deployment = this.modelFor('deployment');
-    return this.store.find('foreman-task', deployment.get('foreman_task_uuid'));
+    return Ember.RSVP.hash({
```

Comment:
@tzumainn, why are you deleting the model for 
```
return this.store.find('foreman-task', deployment.get('foreman_task_uuid'));
```
This gives the progress of the dynflow action